### ISSUE TITLE
Transifex for localization

### ISSUE THREAD

**Issue by jnv** [crewmate/crewmate#136]
I have created an [organization](https://www.transifex.com/organization/crewmate) and [the project](https://www.transifex.com/projects/p/crewmate/) at Transifex, which is (IMO) the leading platform for open-source projects localization. If you have an account at Transifex, either invite yourself in or give me your username, and I will add you to the project.

This is still an early phase support, for example the translation files are not synced back automatically (there is no out-of-the-box solution for Transifex), but I hope this will make things easier for translators.

CC @versvs, @nukeador
Refs: #135

**Comment by pierre-alain-b** [comment]
Hello, I joined the transifex account and started to translate in FR. I am almost done. How can I then put back the i18n work in the crewmate tree?
Thanks !

**Comment by pierre-alain-b** [comment]
Yes, all is done. Thanks for you help to get to the next step and put that in the app!

**Comment by pierre-alain-b** [comment]
Done, I exported the .yml file and created a pull request to have it included in the master tree
